4001-5000/LIT4838CollectioM.xml

```diff
@@ -85,8 +86,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <ab>
                  ቀዳሜ፡ ትእዛዙ፡ ለአብ፡ ዘይቤ፡ ዘቦ፡ እዝነ፡ ለይስማዕ። ይቤ፡ እግዚአብሔር፡ ሰማይ፡ መንበርየ፡
                </ab>
+               <div type="textpart" subtype="chapter" n="1" xml:id="Chapter1" xml:lang="gez">
```

Comment:
I assume that the textparts follow the edition, but it would maybe be better to clarify this